waspc/run

```diff
@@ -78,6 +79,8 @@ print_usage() {
     "Builds the Haskell project."
   print_usage_cmd "build:all" \
     "Builds the Haskell project + all sub-projects (i.e. TS packages)."
+  print_usage_cmd "build:all:static" \
```

Comment:
Let's maybe explain here that this won't work unless musl is present instead of glibc, and also works only on Linux (if that is correct?). Basically let's explain the requirements/conditions, if there are some.